cla-backend-go/signatures/models/models.go

```diff
@@ -1,15 +1,21 @@
 // Copyright The Linux Foundation and each contributor to CommunityBridge.
 // SPDX-License-Identifier: MIT
 
-package signatures
+package models
```

Comment:
Maintain the models.py . You had added the models folder

Reply:
@nickmango this models, i need to use in gitlab_activity, and i can't use signatures package in it due to cyclic dependency.

Reply:
The cyclic issue can be resolved with interfaces @umeshlumbhani247

Reply:
I have updated the code, Please review @nickmango